tests/fixtures/yaml_dicts/index.md

```diff
@@ -0,0 +1,14 @@
+extends: index.liquid
+
+title:   My first Blogpost
+date:    24/08/2014 at 15:36
+some-data:
+  - nested:
+      dicts: Works
+  - nested:
+      dicts: just fine
+---
```

Comment:
This is the functionality I wish to have when using Cobalt. Structuring data in yaml like this is a great tool. If there is a "better" (more idiomatic) way of doing this, please let me know.